enterprise/backup/src/main/java/org/neo4j/backup/BackupCopyService.java

```diff
@@ -82,9 +104,8 @@ private File findAnAvailableBackupLocation( File file, String pattern )
             final AtomicLong counter = new AtomicLong( 0 );
             Consumer<File> countNumberOfFilesProcessedForPotentialErrorMessage = generatedBackupFile -> counter.getAndIncrement();
 
-            return availableAlternativeNames( file, pattern )
-                    .peek( countNumberOfFilesProcessedForPotentialErrorMessage )
-                    .filter( f -> !backupExists(f) )
+            return availableAlternativeNames( file, pattern ).peek( countNumberOfFilesProcessedForPotentialErrorMessage )
+                    .filter( f -> !backupExists( f ) )
```

Comment:
I preferred the old formatting. Likewise lines 118 and 123.

Reply:
Addressed